xdai.json

```diff
@@ -1,94 +1,131 @@
+      "wrappableToken": {
+        "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/Agave/logo.png"
+      },
+      "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/Agave/logo.png",
+      "logo_type": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/Agave/logo_type.png",
+      "forum": "https://forum.1hive.org/",
+      "links": {
+        "community": [
+          {
+            "label": "Discord",
+            "link": "https://discord.gg/86eVaKfv2Z"
+          },
+          {
+            "label": "Github",
+            "link": "https://github.com/agave-dao"
+          },
+          {
+            "label": "Twitter",
+            "link": "https://twitter.com/Agave_lending"
+          },
+          {
+            "label": "Telegram",
+            "link": "https://t.me/Agave1Hive"
+          },
+          {
+            "label": "Forum",
+            "link": "https://forum.1hive.org/"
+          }
+        ],
+        "documentation": [
+          {
+            "label": "FAQs",
+            "link": "https://wiki.1hive.org/guides/agave-faq"
+          }
+        ]
+      }
     },
-    "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/Agave/logo.png",
-    "logo_type": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/Agave/logo_type.png",
-    "forum":"https://forum.1hive.org/",
-    "links": {
-      "community": [
-        {
-          "label": "Discord",
-          "link": "https://discord.gg/86eVaKfv2Z"
-        },
-        {
-          "label": "Github",
-          "link": "https://github.com/agave-dao"
-        },
-        {
-          "label": "Twitter",
-          "link": "https://twitter.com/Agave_lending"
-        },
-        {
-          "label": "Telegram",
-          "link": "https://t.me/Agave1Hive"
-        },
-        {
-          "label": "Forum",
-          "link": "https://forum.1hive.org/"
-        }
-      ],
-      "documentation": [
-        {
-          "label": "FAQs",
-          "link": "https://wiki.1hive.org/guides/agave-faq"
-        }
-      ]
+    {
+      "address": "0x1E2d5fB385E2EAE45Bd42357E426507a63597397",
+      "name": "BrightDAO",
+      "description": "The BrightDAO intends to contribute to the support, development, engineering, and adoption of the BrightID social identity network.",
+      "wrappableToken": {
+        "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/BrightDAO/BrightLogo.png"
+      },
+      "token_logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/BrightDAO/BrightWrappedLogo.png",
+      "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/BrightDAO/BrightLogo.png",
+      "logo_type": "https://raw.githubusercontent.com/1Hive/dao-list/master/assets/BrightDAO/BrightLogo.png",
+      "wiki": "https://brightid.gitbook.io/brightid/",
```

Comment:
let's rename all the assets names following what we have in the other folders, there is no need to add the dao name since we already have the name on the folder